Source/utils/display.cpp

```diff
@@ -121,6 +124,48 @@ void CalculatePreferdWindowSize(int &width, int &height)
 #endif
 }
 
+void CalculateSubPanelArea(void)

```

Comment:
Sub -> Side, but it also does main so:
```suggestion
void CalculatePanelAreas(void)
```

Reply:
I will modify related part and update later.

Reply:
By the way I don't think this really belongs in `utils/display.cpp` as it is part of the game interface and not the computer abstraction. I suggest moving it to `control.cpp`